### PR TITLE
fix(dependabot): add back npm patch upgrades for stable branches

### PR DESCRIPTION
**Pull request by skjnldsv**
Seems like copy/pasting went wrong at some point :'(

oringaly from https://github.com/nextcloud/server/pull/31216 by @blizzz

### CODE REVIEW

File: .github/dependabot.yml [hunk 1]
```diff
@@ -136,7 +136,7 @@ updates:
   ignore:
     # ignore all GitHub linguist patch updates
```
Comment (susnux) [inline .github/dependabot.yml:137]: This comment makes not really sense

**Review by artonge — APPROVED** [review]

File: .github/dependabot.yml [hunk 2]
```diff
@@ -116,7 +116,7 @@ updates:
...
```
Comment (skjnldsv) [inline .github/dependabot.yml:117]:
```suggestion
```

File: .github/dependabot.yml (hunk 1 above)
Comment (skjnldsv) [reply .github/dependabot.yml:137]:
```suggestion
```

**Review by susnux — APPROVED** [review]

**Comment by blizzz** [comment]
> oringaly from #31216 by @blizzz

There the changes applied only to an EOL branch. Not against stable ones, as they are now. Blind copy and paste, or blind overwriting of branch names?

**Pull request merged**